### PR TITLE
Fixes #2957 Endothelial surface area "Blood flow" - wrong values in the simulation

### LINKED ISSUE #2957: Endothelial surface area "Blood flow" - wrong values in the simulation?
The values AND formulas differ between the Individual and the actual simulation.

- Individual with "Endothelial surface area" calculation "Organ vascularization":
image

Uses the correct equation:
image

- Individual with "Endothelial surface area" calculation "Blood flow":
 
image

Uses the correct equation, mind that the values are different (as expected):

image

- Simulation using the individual with "Organ vascularization" - correct values (as in the individual):

image

## !!!!!
Simulation using the individual with "Blood flow" - WRONG formula AND the values are different from those in the individual!!

image

### PR DESCRIPTION
**Pull request by rwmcintosh**
Fixes #2957

# Description
Removing the update of ModelProperties

## Type of change

Please mark relevant options with an `x` in the brackets.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires documentation changes (link at least one [user](https://github.com/Open-Systems-Pharmacology/docs) or [developer](https://github.com/Open-Systems-Pharmacology/developer-docs) documentation issue):
- [ ] Algorithm update - updates algorithm documentation/questions/answers etc.
- [ ] Other (please describe):

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Integration tests
- [x] Unit tests
- [x] Manual tests 
- [ ] No tests required

# Reviewer checklist

Mark everything that needs to be checked before merging the PR.

- [ ] Check if the code is well documented
- [ ] Check if the behavior is what is expected
- [ ] Check if the code is well tested
- [ ] Check if the code is readable and well formatted
- [ ] Additional checks (document below if any)
- [ ] Check if documentation update issue(s) are created if the option `This change requires a documentation update` above is selected

# Screenshots (if appropriate):

# Questions (if appropriate):

### CODE REVIEW

**Comment by rwmcintosh** [comment]
It's very tough for me to tell if this is the right approach. It appears that the concept of updating calculation methods from a new MoleculeProperties and old MoleculeProperties is not needed now.

I tried using different species and individuals with different calculation methods and it seems like we just want to use the newly created default MoleculeProperties in all these cases.

As pointed out during our dev meeting by @msevestre this code pre-dates Open Sourcing, so there's not much history available on it.

File: src/PKSim.Presentation/Presenters/Simulations/SimulationModelSelectionPresenter.cs [hunk 1]
```diff
@@ -59,7 +59,7 @@ public IEnumerable<ModelConfiguration> AllModels()
       public void ModelSelectionChanging(ModelConfiguration newModelConfiguration)
       {
          ModelConfiguration = newModelConfiguration;
-         _modelPropertiesToUse = _modelPropertiesTask.Update(ModelProperties, _modelPropertiesTask.DefaultFor(newModelConfiguration, _originData), _originData);
+         _modelPropertiesToUse = _modelPropertiesTask.DefaultFor(newModelConfiguration, _originData);
```
Comment (rwmcintosh) [inline src/PKSim.Presentation/Presenters/Simulations/SimulationModelSelectionPresenter.cs:62]: In a nutshell here - do we need to combine ModelProperties from old to new?

File: tests/PKSim.Tests/IntegrationTests/ModelPropertiesTaskSpecs.cs [hunk 2]
```diff
@@ -30,7 +30,7 @@ public class When_updating_model_properties : concern_for_ModelPropertiesTask
 
       protected override void Because()
       {
-         _updatedProperties = sut.Update(_modelProperties_4Comp, _modelProperties_2Pores, _originData);
+         _updatedProperties = _modelProperties_2Pores;
```
Comment (rwmcintosh) [inline tests/PKSim.Tests/IntegrationTests/ModelPropertiesTaskSpecs.cs:33]: depending the need for the concept of Update, this test should go

File: src/PKSim.Presentation/Presenters/Simulations/SimulationModelSelectionPresenter.cs (hunk 1 above)
Comment (msevestre) [reply src/PKSim.Presentation/Presenters/Simulations/SimulationModelSelectionPresenter.cs:62]: this is what was us causing the problem. Creating new one from scratch should be good I believe
Every entry that was set by hand will be reset but this seems to be ok
Do we need the .Update method at all? if not, I would just remove it

Comment (rwmcintosh) [reply src/PKSim.Presentation/Presenters/Simulations/SimulationModelSelectionPresenter.cs:62]: yeah, it's removed in this PR. I'll remove the tests for it and make the PR ready for review

**Pull request merged**